docs/developer/procedures/git.rst

```diff
@@ -103,7 +103,7 @@ Feature Branches
 
 To work effectively with Git one must embrace the notion of branch development. Taking
 this to the extreme would mean that a developer should never do any work on a primary 
-branch. Instead creating a *feature* branch, performing a change, and then merging the
+branch. Instead create a *feature* branch, performing a change, and then merging the
```

Comment:
You forgot to change the tense of the rest of the sentence. Should be:

Instead create a *feature* branch, perform a change, and then merge the